cfme/tests/test_login.py

```diff
@@ -12,6 +12,35 @@
 pytestmark = pytest.mark.usefixtures('browser')
 
 
+def new_cred(username, password):
+    username = username or conf.credentials['default']['username']
+    password = password or conf.credentials['default']['password']
+    cred = Credential(principal=username, secret=password)
+    return cred
+
+
+def new_user_instantiate(appliance, username=None, password=None, name=None):
+    cred = new_cred(username=username, password=password)
+    name = name or 'Administrator'
+    user = appliance.collections.users.instantiate(credential=cred, name=name)
+    return user
+
+
+def new_user_create(appliance, request, username=None, password=None, group=None, name=None):
+    cred = new_cred(username=username, password=password)
+    user_group = appliance.collections.groups.instantiate(
+        description=group or "EvmGroup-vm_user"
+    )
+    name = name or username
```

Comment:
both of these could be None, you handle the default username when creating the `Credential` instance though, so you can use it here to default the name:
` name = name or cred.principal`